homeassistant/components/loqed/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "flow_title": "LOQED Touch Smartlock setup",
+    "step": {
+      "user": {
+        "description": "Login at {config_url} and: \n* Create an API-key by clicking 'Create' \n* Copy the created access token.",
```

Comment:
My feedback would be to just change this to "Login at Loqed and create an API-key....."